modules/common/src/main/scala/surge/streams/EventSink.scala

```diff
@@ -28,3 +36,11 @@ trait EventSink[Event] extends EventHandler[Event] {
     FlowConverter.flowFor(handleEvent, partitionBy, sinkExceptionHandler, parallelism)(ExecutionContext.global)
   }
 }
+
+trait EventSinkSupport[Event] {
+  def metrics: Metrics = Metrics.globalMetricRegistry
+  def metricTags: Map[String, String] = Map("class" -> getClass.getCanonicalName)
+  def formatMetaForExceptionHandlerMessage[Meta]: EventPlusStreamMeta[String, Event, Meta] => String = _.streamMeta.toString
```

Comment:
It seems like we could just add these directly to `EventHandler`. Metrics stuff seems like it should be available for any `EventHandler` as we add more observability going forward and an `EventHandler` that does not extend `EventSinkSupport` would get the same implementation as this default anyways via the `DefaultDataSinkExceptionHandler`.

Reply:
Chatted live on this and we'll keep it the way it is to preserve backwards compatibility for now at least. We can look at if it makes sense to revisit later